.github/workflows/export.yml

```diff
@@ -64,8 +66,61 @@ jobs:
           # binary location
           ./godot --headless --verbose --path ../ --export-release "Web" ./build/web/index.html
 
-      - name: Upload Artifact
+      - name: Build / PCK
+        run: |
+          mkdir -v -p build && cd build
+
+          # Note that the export path can be confusing; it's relative to the
+          # Godot project path, NOT necessarily the current directory or Godot
+          # binary location
+          ./godot --headless --verbose --path ../ --export-pack "Linux" ./build/${PCK_NAME}.pck
```

Comment:
Is the `.pck` file actually different between the web and Linux builds?

If yes, then the web & Linux builds are separate from one another so they could be in different jobs, and so run in parallel.

If no, then this step is not necessary: just upload the `.pck` file that was generated by the web build as a standalone artefact so it can be easily used by the `flatpak` job.

Reply:
Confirmed by exporting a PCK from the editor with both the web and Linux presets. And then comparing with the PCK exported from the regular web export for good measure:

```
$ sha256sum web.pck && sha256sum linux.pck && sha256sum web/index.pck 
53b40a38886eaa2e7e900dd192bfd0d4ba135141d2ab7d9073b8117dde54f233  web.pck
53b40a38886eaa2e7e900dd192bfd0d4ba135141d2ab7d9073b8117dde54f233  linux.pck
53b40a38886eaa2e7e900dd192bfd0d4ba135141d2ab7d9073b8117dde54f233  web/index.pck
```